components/brave_education/common/education_content_urls.cc

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_education/common/education_content_urls.h"
+
+#include <map>
+#include <utility>
+
+#include "base/containers/fixed_flat_map.h"
+#include "brave/components/constants/webui_url_constants.h"
+
+namespace brave_education {
+
+namespace {
+
+constexpr std::string_view TypeToBrowserURL(EducationContentType content_type) {
```

Comment:
see below re: naming, but also again please use consistent naming. We have `EducationContentTypeFromBrowserURL` and `TypeToBrowserURL`. Also why is this a string and not a GURL?

Reply:
Renamed for consistency. `std::string_view` is used because these are `constexpr`.